lib/t/application.ex

```diff
@@ -28,11 +28,11 @@ defmodule T.Application do
         TWeb.Presence,
         TWeb.UserSocket.Monitor,
         T.Repo,
+        maybe_migrator(),
```

Comment:
migrations need to run before oban since they create/update tables for it, although it doesn't matter anymore since the tables are already created